### ISSUE TITLE
Training_Set Validation_Set Test_Set

### ISSUE THREAD

**Issue by zasr99** [xufangchn/GLF-CR#14]
Could you please tell me how the training set, validation set and test set are divided?Thanks！
